.github/workflows/black.yml

```diff
@@ -28,7 +28,7 @@ jobs:
 
       # install black (extract version from versions.cfg)
       - name: install black
-        run: pip install black==$(awk '/^black =/{print $NF}' versions.cfg)
+        run: pip install click==8.0.4 black==$(awk '/^black =/{print $NF}' versions.cfg)
```

Comment:
@ericof I'd prefer to pin versions only in versions.cfg and get them from versions.cfg (like we do for the black version). Otherwise it is very hard to track different versions over time.